docs/source/algorithms.rst

```diff
@@ -50,3 +50,152 @@ Using the same example file as before, we can execute a GaussFilter on our test
     gf.filterExperiment(exp)
     # MzMLFile().store("test.filtered.mzML", exp)
 
+
+
+Spectra Merge Algorithm
+*************************
+
+Spectra merge algorithms merge several spectra to e.g., improve spectrum quality by increasing S/N ratio.
+Merging is usually done block wise (for MS1 and above) or (for MS2 and above) based on similar precursors.
+In any case, the number of scans will be reduced. Some methods allow to smooth or average spectra. 
```

Comment:
is this true? you show examples below were it doesn't get reduced and the averaging also doesn't seem to reduce the number of spectra